custom_components/tuya_local/devices/point_one_storm_vspec_iv_dual_axis_corner_fan.yaml

```diff
@@ -0,0 +1,98 @@
+name: Corner fan
+products:
+  - id: lzjzcjqgzwkt6ked
+    manufacturer: Point One Technology
+    model: Storm VSPEC-IV Dual-axis Smart Corner Fan
+entities:
+  - entity: light
+    dps:
+      - id: 108
+        type: boolean
+        name: Switch
+      - id: 22
+        type: integer
+        name: Brightness
+        range:
+          min: 10
+          max: 1000
+      - id: 23
+        type: integer
+        name: Temperature
+        range:
+          min: 0
+          max: 1000
+  - entity: fan
+    dps:
+      - id: 101
+        type: boolean
+        name: Switch
+      - id: 102
+        type: string
+        name: Mode
+        mapping:
+          - dps_val: nature
+            value: nature
+          - dps_val: sleep
+            value: sleep
+          - dps_val: fresh
+            value: fresh
+          - dps_val: strong
+            value: strong
+      - id: 103
+        type: integer
+        name: Speed
+        range:
+          min: 1
+          max: 6
+      - id: 104
+        type: boolean
+        name: Oscillate vertically
+      - id: 105
+        type: boolean
+        name: Oscillate horizontally
+      - id: 107
+        type: boolean
+        name: Fan sound
+      - id: 112
+        type: boolean
+        name: Rotate 360°
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 109
+        type: string
+        name: Countdown set time
+        mapping:
+          - dps_val: "cancel"
+            value: "cancel"
+          - dps_val: "1h"
+            value: "1h"
+          - dps_val: "2h"
+            value: "2h"
+          - dps_val: "3h"
+            value: "3h"
+          - dps_val: "4h"
+            value: "4h"
+          - dps_val: "5h"
+            value: "5h"
+          - dps_val: "6h"
+            value: "6h"
+      - id: 110
+        type: integer
+        name: countdown_time_left
+        range:
+          min: 0
+          max: 360
+        unit: min
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 111
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
```

Comment:
Similar to above: This is a read-only value so I'm not sure if the implementation will be similar to the other properties/values above.

Reply:
> And if there are meaningful fault codes already known, you can also add them as a `description` attribute

This is nifty! Unfortunately, I can't find any information on the fault code after looking through the product documentation. 😬

Reply:
Does that mean I should remove the `mapping:` block?

Reply:
I've done that in [`beb49a6` (#3427)](https://github.com/make-all/tuya-local/pull/3427/commits/beb49a6c854dc0d5a1c88606506b54d31c43231f). Feel free to let me know if this entity requires any further changes. 🙏